examples/otel-tracing/README.md

```
@@ -0,0 +1,117 @@
+  -p 16686:16686 \
+  -p 4318:4318 \
+  jaegertracing/all-in-one:latest
+```
+
+2. Run eval with OTLP export:
+
+```bash
+OTEL_EXPORTER_OTLP_ENDPOINT=http://localhost:4318 npx promptfoo eval
+```
+
+3. View at http://localhost:16686
```

markdownlint-cli2 (0.18.1)

Bare URL used

(MD034, no-bare-urls)